subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -204,6 +204,7 @@ config BT_CTLR_SCAN_SYNC_ISO_SET
 config BT_CTLR_ZLI
 	bool "Use Zero Latency IRQs"
 	depends on ZERO_LATENCY_IRQS
+	default y
```

Comment:
Why not `default ZERO_LATENCY_IRQS`?

Reply:
1. Cortex M0 and other architectures do not support `CONFIG_ZERO_LATENCY_IRQS`
2. Increases ISR Stack Size due to additional nesting
3. CONFIG_PM=y disables all interrupts